formats/kubevirt.nix

```diff
@@ -0,0 +1,20 @@
+{ config, lib, pkgs, modulesPath, ... }:
+{
+  imports = [
+    "${toString modulesPath}/virtualisation/kubevirt-config.nix"
+  ];
+
+  system.build.kubevirtImage =  import "${toString modulesPath}/../lib/make-disk-image.nix" {
+    inherit lib config pkgs;
+    diskSize = 8192;
```

Comment:
if you leave the diskSize empty it should automatically detect the required space

Reply:
removed!